oak-benchmarks-elastic/src/main/java/org/apache/jackrabbit/oak/benchmark/ElasticFullTextWithGlobalIndexSearchTest.java

```diff
@@ -54,8 +54,8 @@ public ElasticFullTextWithGlobalIndexSearchTest(File dump, boolean flat, boolean
                 @Override
                 public Jcr customize(Oak oak) {
                     ElasticsearchIndexEditorProvider editorProvider = new ElasticsearchIndexEditorProvider(coordinate,
-                            new ExtractedTextCache(10 * FileUtils.ONE_MB, 100));
-                    ElasticsearchIndexProvider indexProvider = new ElasticsearchIndexProvider(coordinate);
+                            new ExtractedTextCache(10 * FileUtils.ONE_MB, 100), null);
+                    ElasticsearchIndexProvider indexProvider = new ElasticsearchIndexProvider(coordinate, null);
```

Comment:
It would be better if we actually use a prefix here instead of null, these benchmarks tests would be executed on a single ES server and would act as multiple tenants - so would make sense to pass a legit prefix here.

Also, adding some randomization to the prefix would be required (See comment in ElasticsearchIndexProviderService). 

Otherwise we would need some mechanism to delete the index from elastic server after each test run (I don't think currently we have a delete mechanism exposed in the client - if we do have, that could be another problem).

We can handle the cleanup externally too (shell script calling index delete curl req) - but I would like that to happen after the complete job is run (all the benchmark tests) - so I think it would be good to handle where multiple tests can run on the same ES server without having the need to delete the underlying index after every test run.